src/test/kotlin/org/rust/ide/inspections/RsUnresolvedReferenceInspectionTest.kt

```diff
@@ -212,6 +212,15 @@ class RsUnresolvedReferenceInspectionTest : RsInspectionsTestBase(RsUnresolvedRe
         extern crate unknown_crate2;
     """)
 
+    @MinRustcVersion("1.56.0")
```

Comment:
```suggestion
    @MinRustcVersion("1.56.0-nightly")
```
since `1.56.0-nightly` < `1.56.0`